### PR TITLE
Add Bert-Large and T5-Large TF models in various batch sizes

### PR DESCRIPTION
**Pull request by mariecwhite**
Bert-Large and T5-Large are part of our progression suite that we will be monitoring closely at various batch sizes.

### CODE REVIEW

File: .github/workflows/benchmark_long.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ on:
     # Scheduled to run at 09:00 UTC and 21:00 UTC.
     - cron: '0 09,21 * * *'
   workflow_dispatch:
+  pull_request:
```
Comment (GMNGeoffrey) [inline .github/workflows/benchmark_long.yml]: Remember to remove

File: .github/workflows/build_e2e_test_artifacts.yml [hunk 2]
```diff
@@ -96,6 +96,10 @@ jobs:
         run: gcloud storage cp "${TF_BINARIES_GCS_ARTIFACT}" "${TF_BINARIES_ARCHIVE}"
       - name: "Extracting TF binaries archive"
         run: tar -xf "${TF_BINARIES_ARCHIVE}"
+      - name: "Enlarge disk"
+        run: |
+          sudo mount -o remount,size=200G /runner-root
```
Comment (GMNGeoffrey) [inline .github/workflows/build_e2e_test_artifacts.yml]: Ah, so this is tricky. That disk is a RAM disk. The whole machine has 360GB of RAM, so this is using more than half of it. Are you sure that's what we want?

Generally I think we probably don't want workflows messing with the disk. I've held off on switching this to a local SSD like we have for the GPU machines because the n1-standard-96 requires having something like 24 local SSD if you have any, which is a bit much. We could also just try switching this to having persistent SSD on the side.

Comment (pzread) [reply .github/workflows/build_e2e_test_artifacts.yml]: Chatted offline with @GMNGeoffrey, we will try to find another way to get enough disk space (maybe persisted SSD)

The workflow run shows that we need 199GB currently, so we probably need at least 250GB of disk space. If we do this on RAM disk, it's a little bit too much for the total 360GB of RAM on the build machines.

Comment (mariecwhite) [reply .github/workflows/build_e2e_test_artifacts.yml]: Thanks for the update

**Comment by mariecwhite** [comment]
I suggest uncommenting this line: https://github.com/openxla/iree/blob/main/.github/workflows/run_shark_tank.yml#L10 to test the SHARK workflow during presubmit.

File: .github/workflows/build_e2e_test_artifacts.yml (hunk 2 above)
Comment (GMNGeoffrey) [reply .github/workflows/build_e2e_test_artifacts.yml]: https://github.com/openxla/iree/pull/13079 has been rolled out and all the runners should have 1TB of disk now

File: build_tools/python/benchmark_suites/iree/cuda_benchmarks.py [hunk 3]
```diff
@@ -24,7 +24,10 @@ class Linux_CUDA_Benchmarks(object):
   SM_80_COMPILE_CONFIG = iree_definitions.CompileConfig.build(
       id=unique_ids.IREE_COMPILE_CONFIG_LINUX_CUDA_SM80_DEFAULTS,
       tags=["default-flags"],
-      compile_targets=[SM_80_GPU_TARGET])
+      compile_targets=[SM_80_GPU_TARGET],
+      extra_flags=[
+          "--iree-codegen-llvmgpu-use-mma-sync",
```
Comment (pzread) [inline build_tools/python/benchmark_suites/iree/cuda_benchmarks.py]: These are the benchmarks with default flags, so it looks a little bit weird to set flags here. What is this flag doing and is it possible to make this option enabled by default?

Comment (mariecwhite) [reply build_tools/python/benchmark_suites/iree/cuda_benchmarks.py]: The plan is to enable this functionality by default but we are currently running into some compiler errors for some models: https://github.com/openxla/iree/issues/13075

Comment (pzread) [reply build_tools/python/benchmark_suites/iree/cuda_benchmarks.py]: Maybe add a TODO with reason?

File: build_tools/python/e2e_test_framework/definitions/iree_definitions_test.py
Comment (pzread) [inline build_tools/python/e2e_test_framework/definitions/iree_definitions_test.py:1]: Why is this file changed?

**Review by pzread — APPROVED** [review]

File: build_tools/python/benchmark_suites/iree/cuda_benchmarks.py (hunk 3 above)
Comment (mariecwhite) [reply build_tools/python/benchmark_suites/iree/cuda_benchmarks.py]: Added a comment

File: build_tools/python/e2e_test_framework/definitions/iree_definitions_test.py
Comment (mariecwhite) [reply build_tools/python/e2e_test_framework/definitions/iree_definitions_test.py:1]: Good catch. I removed the changes.

File: build_tools/python/benchmark_suites/iree/cuda_benchmarks.py (hunk 3 above)
Comment (GMNGeoffrey) [reply build_tools/python/benchmark_suites/iree/cuda_benchmarks.py]: No we don't want extra flags in "default flags" configs. That is the whole reason for them. I can dig up the PR where I created this split. If this still needs some special flags to run then it should be in "experimental flags".

**Review by GMNGeoffrey — CHANGES_REQUESTED** [review]

File: build_tools/python/benchmark_suites/iree/cuda_benchmarks.py (hunk 3 above)
Comment (mariecwhite) [reply build_tools/python/benchmark_suites/iree/cuda_benchmarks.py]: That's fair. I'll let the relevant parties know that it won't be enabled by default until the compiler issues are fixed.

**Review by GMNGeoffrey — APPROVED** [review]
